app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt

```diff
@@ -6,11 +6,33 @@
 
 package mozilla.lockbox.presenter
 
+import mozilla.lockbox.adapter.SectionedAdapter
+import mozilla.lockbox.adapter.SettingCellConfiguration
+import mozilla.lockbox.adapter.TextSettingConfiguration
+import mozilla.lockbox.adapter.ToggleSettingConfiguration
 import mozilla.lockbox.flux.Presenter
 
-interface SettingViewProtocol
+interface SettingView {
+    fun updateSettingList(
+        settings: List<SettingCellConfiguration>,
+        sections: List<SectionedAdapter.Section>
+    )
+}
 
-class SettingPresenter(view: SettingViewProtocol) : Presenter() {
+class SettingPresenter(val view: SettingView) : Presenter() {
     override fun onViewReady() {
+        val settings = listOf(
+            ToggleSettingConfiguration("Unlock with fingerprint", toggle = false),
+            TextSettingConfiguration("Auto lock", detailText = "5 minutes"),
+            ToggleSettingConfiguration("Autofill", "Let Firefox Lockbox fill in logins for you", false),
+            ToggleSettingConfiguration("Send usage data", "Mozilla strives to only collect what we need to provide and improve Firefox for everyone. ", true)
```

Comment:
(Yes, I realize this is still WIP)

When putting in the TelemetryStore basics, included resource strings/bools in place.  Is it reasonable to hook that in here now?

Reply:
#120 is to record Telemetry events around settings, not hooking in the "send usage data" setting

Reply:
Oh, you're referring to story #8

Reply:
Yes, that's the one!  Might be nice to sneak it in now anyway, but please carry on! (-: